.github/workflows/flow-diagram-explorer.yml

```diff
@@ -34,9 +34,9 @@ jobs:
               run: |
                   npm run typecheck && npm run lint
 
-            - name: 🤖 Run tests
-              run: |
-                  npm run test
+            #- name: 🤖 Run tests
```

Comment:
Should this be added again?

Reply:
Yes, as soon as we have tests. Since we don't have any yet, `jest` would always fail.